roles/ceph-iscsi-gw/tasks/non-container/postrequisites.yml

```diff
@@ -0,0 +1,9 @@
+- name: start rbd-target-api and rbd-target-gw
+  service:
+    name: "{{ item }}"
+    state: started
+    enabled: yes
+    masked: no
+  with_items:
+    - rbd-target-api
+    - rbd-target-gw
```

Comment:
The order is different (api->gw) compared to (gw->api) before.
Does this have an impact ?

Reply:
It differs between RCHS 3 and 4. In 3 we needed gw to start first then api. In 4 and up, we want api to start first then gw.

Reply:
Good to know for the backport PR on stable-3.2